### DISCUSSION TITLE
Welcome to sec-edgar Discussions!

### DISCUSSION THREAD

**Discussion by jackmoody11** [sec-edgar/sec-edgar#146] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Elmacaroon** [comment]
Hey, I'm Jan. I'm an Economics policy student from Czechia and I've stumbled upon this crawler in my search for a way how to download the SEC Edgar filings for space companies! I just wanted to say that I'm extremely grateful that you created this crawler because I was struggling quite a bit with my scraping program!

**Reply by jackmoody11** [reply]
Hi, Jan. Glad you found it helpful! Thanks for the kind words.

**Comment by ehaynie63** [comment]
![Screenshot (10)](https://github.com/user-attachments/assets/1de4af0e-7544-4b4d-b480-97c4e70ede23)
Hi, I'm getting a json decode error when trying to run the example in the readme. 

Can anyone provide advice on how to resolve this.

Thanks!
